workspace/assets/js/dev/panel.js

```diff
@@ -158,10 +158,15 @@
 		_.each(checkboxes, function (c) {
 			inner.append(c);
 		});
+
 		var attached = false;
 
 		$(document).on('keydown', function (e) {
-			if (e.which === global.keys.escape) {
+			if (e.which === global.keys.escape && attached) {
+				var attFx = attached ? 'detach' : 'appendTo';
+				panel[attFx](attached ? undefined : body);
+				attached = !attached;
+			} else if (e.which === global.keys.pause_break && e.shiftKey) {
 				var attFx = attached ? 'detach' : 'appendTo';
 				panel[attFx](attached ? undefined : body);
 				attached = !attached;
```

Comment:
Same here.

Reply:
Its the toggle part. may open or close the popup...

Reply:
ah ok, nvm.